src/gui/filedetails/FileDetailsPage.qml

```diff
@@ -81,7 +81,19 @@ Page {
             Layout.topMargin: root.topPadding
 
             columns: root.showCloseButton ? 3 : 2
-            rows: showFileLockedString ? 3 : 2
+            rows: {
+                let rows = 2;
+
+                if (showFileLockedString) {
+                    rows++;
+                }
+
+                if (root.fileDetails.fileTagModel.totalTags > 0) {
```

Comment:
@claucambra can no you have one if? `if (showFileLockedString || root.fileDetails.fileTagModel.totalTags > 0)`?